src/ucp/wireup/address.c

```diff
@@ -325,8 +325,11 @@ static size_t ucp_address_packed_value_size(size_t value, size_t max_value,
 {
     if (addr_version == UCP_OBJECT_VERSION_V1) {
         /* Address version 1 does not support value extension */
-        ucs_assertv_always(value <= max_value, "value %zu, max_value %zu",
-                           value, max_value);
+        if (value > max_value) {
+            ucs_fatal("value %zu > max_value %zu, use UCX_ADDRESS_VERSION=v2 to"
```

Comment:
does it have to be a fatal error, or can we propagate the failure to UCP API?

Reply:
fixed, now:
```
address.c:331  UCX  DEBUG value 18 > max_value 31, use UCX_ADDRESS_VERSION=v2 to allow packing more addresses
libperf.c:1251 UCX  ERROR ucp_worker_get_address() failed: Unsupported operation
libperf.c:2043 UCX  ERROR ucp_perf_test_setup_endpoints() failed: Unsupported operation
```